manifests/scripts/metrics-dashboard/run.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+# -----------------
+
+# Using OpenShift's Prometheus instance requires us to enable user workload monitoring via the openshift-monitoring ConfigMap:
+# https://docs.openshift.com/container-platform/4.11/monitoring/enabling-monitoring-for-user-defined-projects.html)
+# - BUT, the way I'm doing it here is a brittle hack :P
+kubectl apply -f prometheus/enable-user-workload-monitoring.yaml
+
+# Allow Prometheus process to view the openshift-operators/gitops namespaces
+kubectl apply -f prometheus/prometheus-roles-for-openshift-operators.yaml -n openshift-operators
+kubectl apply -f prometheus/prometheus-roles-for-gitops-ns.yaml -n gitops
+
+# -----------------
+echo 
+echo "* Kick off an OLM install of Grafana, and wait for it to complete"
+
+kubectl create ns grafana > /dev/null 2>&1
+
+kubectl apply -f grafana/grafana-operator-group.yaml -n grafana
+kubectl apply -f grafana/grafana-subscription.yaml -n grafana
+
+echo
+echo "* Waiting for Grafana CRDs to exist"
+
+while : ; do
+  kubectl get customresourcedefinition/grafanas.integreatly.org  > /dev/null 2>&1 && break
+  sleep 1s
+done
+
+# -----------------
+
+echo "* Waiting for Grafana route"
+
+while : ; do
```

Comment:
When I ran the script, the grafana route didn't get created automatically.  I had to create it manually:
```
oc create route edge grafana-route -n grafana --service grafana-operator-controller-manager-metrics-service
```

Reply:
Hmm, not sure how it was working before :thinking:.